src/main/java/frc/lib/wrappers/motorcontrollers/NomadTalonSRX.java

```diff
@@ -22,7 +32,9 @@
     protected boolean manualFollowing = false;
     protected double lastLeaderOutput = Double.NaN;
     protected double currentLeaderOutput = Double.NaN;
-
+    /**

```

Comment:
Fixed. I do have several other stuff I left as comments because it's a "recommended implementation" (things like creating drive kinematics using the track width), and I wanted to keep it with the parent class's file. That one was a stray from an experiment I did.

Reply:
> I tried to paste the full thing below, but it doesn't want to format righ

You can use triple backticks ``` to format as code btw.